docs/docs/rest-api/public/api/v2/types/pod-instance-status.raml

```diff
@@ -9,10 +10,8 @@ properties:
       TODO(jdef) Probably represents the Mesos executor ID.
   status:
     type: string
-    usage: |
-      TODO(jdef) this is not an enum because it reflects Marathon internal state that
-      could easily change going forward and I don't want breaking API changes that result
-      from a shifting enum here.
+    (scalaType): PodInstanceState
+    enum: [ PENDING, STAGING, STABLE, DEGRADED, TERMINAL ]
```

Comment:
Did you intentionally add these back? I had removed them because I thought we were just going to dump the string value of the internal pod instance status here (and I didn't want to reflect the internal states as an API enum)